security/pkg/pki/ca/ca.go

```diff
@@ -196,6 +197,33 @@ func NewPluggedCertIstioCAOptions(certChainFile, signingCertFile, signingKeyFile
 		DefaultCertTTL: defaultCertTTL,
 		MaxCertTTL:     maxCertTTL,
 	}
+	if _, err := os.Stat(signingKeyFile); err != nil {
```

Comment:
looks from `createIstioCA` this is handled by `NewSelfSignedIstioCAOptions`

Reply:
Yes - a bit of a mess. I'll try to clean it up further.